sample_data_config/web-api/draft-web-api-s2-embeddings.config.yaml

```diff
@@ -5,18 +5,27 @@ webApi:
     dataset: '{ENV}'
     table: test_s2_specter_v1_embeddings
     source:
-      bigQuery:
-        projectName: 'elife-data-pipeline'
-        sqlQuery:
-          SELECT
-            europepmc_response.doi AS paper_id,
-            europepmc_response.title_with_markup AS title,
-            europepmc_response.abstractText AS abstract,
-          FROM `elife-data-pipeline.prod.v_latest_europepmc_preprint_servers_response` AS europepmc_response
-          WHERE europepmc_response.doi IN (
-            '10.1101/2020.07.24.219204', '10.1101/2023.03.30.534978', '10.1101/2022.08.24.504515',
-            '10.1101/2022.09.28.509871'
-          )
+      include:
+        bigQuery:
+          projectName: 'elife-data-pipeline'
+          sqlQuery:
+            SELECT
+              europepmc_response.doi AS paper_id,
+              europepmc_response.title_with_markup AS title,
+              europepmc_response.abstractText AS abstract,
+            FROM `elife-data-pipeline.prod.v_latest_europepmc_preprint_servers_response` AS europepmc_response
+            WHERE europepmc_response.doi IN (
+              '10.1101/2020.07.24.219204', '10.1101/2023.03.30.534978', '10.1101/2022.08.24.504515',
+              '10.1101/2022.09.28.509871'
+            )
+      exclude:
+        bigQuery:
+          ignoreNotFound: true
+          projectName: 'elife-data-pipeline'
+          sqlQuery:
+            SELECT response.paper_id
+            FROM `elife-data-pipeline.{ENV}.test_s2_specter_v1_embeddings` AS response
+        keyFieldName: 'paper_id'
```

Comment:
We talked about the `keyFieldName` being within `exclude`. I explained that by being required when `exclude` is configured. However, you pointed out that it relates to the field name produced by the `include` query. So maybe could do with a better name or more thinking.